okhttp/src/main/java/okhttp3/internal/http/OneShotRequestBody.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2016 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3.internal.http;
+
+public interface OneShotRequestBody {
```

Comment:
Yes, I like that name a lot better. The other problem with OneShotRequestBody is I kept typing `OSRB` into IntelliJ and getting `OutputStreamRequestBody` !